### PR TITLE
Zeus Docs update

### PR DESCRIPTION
**Pull request by Darth-Coin**
This is just a preliminary PR, to see if is working well the process.
I used Visual Studio to edit the branch and do the PR.
Working on introduction chapter about LN and instructions for connecting various nodes with Zeus.

### CODE REVIEW

**Comment by Darth-Coin** [comment]
Seems that is hanging into uploading the new image file.
How should I proceed with images? Should I upload them manually first and later create the link in the document?

**Comment by kaloudis** [comment]
@Darth-Coin try making a commit for the image itself first before referencing it in a future commit.

Thanks! This is looking great so far.

File: docs/for-users/getting-started/lightning-node-connect.md [hunk 1]
```diff
@@ -7,69 +7,278 @@ import TabItem from '@theme/TabItem';
...
+
+Optional: if you do not want to use Tailscale, the alternative is [ZeroTier.com](https://zerotier.com) (works the same as Tailscale) or [WireGuard](https://wireguard.com) (more advanced).
 
 </TabItem>
 <TabItem value="Raspiblitz">
 
-> iOS developers may be familiar with this concept.
+###Connect Raspiblitz with Zeus
+
+Raspiblitz is using CLN and LND implementations.
```
Comment (kaloudis) [inline docs/for-users/getting-started/lightning-node-connect.md:61]: /is using/supports/

**Comment by Darth-Coin** [comment]
@kaloudis the side bar is not working properly with tabs.
Maybe we could use collapsible sections instead of tabs?
Then the sidebar can use the headings and also linking to the specific chapter/ section.

Example:

<details>
  <summary>Click me</summary>
  
  ### Heading
  1. Foo
  2. Bar
     * Baz
     * Qux

  ### Some Code
  ```js
  function logSomething(something) {
    console.log('Something', something);
  }
  ```
</details>

**Pull request merged**